docs/modules/ROOT/pages/extensions.adoc

```diff
@@ -83,6 +83,77 @@ Upstream best practices and coding conventions for downstream compatibility.
 
 Any extensions missing? Send us a Pull Request!
 
+== Configuring an Extension
+
+If you're writing an extension, you can tie some of its configuration into RuboCop.
+
+=== Config Obsoletions
+When a cop that has been released is later renamed or removed, or one of its parameters is, RuboCop can output error messages letting users know to update their configuration to the newest values. If any obsolete configurations are encountered, RuboCop will output an error message and quit.
+
+You can tie your extension into this system by creating your own `obsoletions.yml` file and letting RuboCop know where to find it:
+
+[source,ruby]
+----
+RuboCop::ConfigObsoletion.files << filename
```

Comment:
Does it make sense to make it clear that it accepts an absolute path?
From extension's code, it might not be obvious that `File.expand_path` should be called.

Reply:
That's a good point, I'll update.